.github/workflows/terraform.yml

```diff
@@ -43,17 +43,19 @@ jobs:
           script: |
             const output = `#### Terraform Format and Style 🖌\`${{ steps.fmt.outcome }}\`
             #### Terraform Initialization ⚙️\`${{ steps.init.outcome }}\`
+            #### Terraform Validation 🤖\`${{ steps.validate.outputs.stdout }}\`
```

Comment:
This will fail if you don't include the following step to this workflow.

```
- name: Terraform Validate
  id: validate
  run: terraform validate -no-color
```